tests/test_data/car.pov

```diff
@@ -16170,39 +16170,39 @@ matrix <2.000, 0.000, 0.000,
 #declare scene = union {
 object {
 part4315
-  pigment { color rgb <0.0, 0.1, 0.1> }
```

Comment:
there's probably a bug somewhere about the color here, this PR is not at fault, the snapshot is :-)